### ISSUE TITLE
Reactive_BLE example not working,not connecting to device discovered

### ISSUE THREAD

**Issue by thusithadeepal9356** [PhilipsHue/flutter_reactive_ble#556]
**Describe the bug**
I used the example https://github.com/PhilipsHue/flutter_reactive_ble/tree/master/example/
**To Reproduce**
Discover devices MacBook pro and Huawei Y9 phone,but click connect, connecting appear for long time,not connecting to device

**Expected behavior**
Need to connect the device.

**Smartphone / tablet**
 - Device: [e.g.Huawei Y9 2018]
 - OS: [Macbook pro]
 - Package version: [flutter_reactive_ble: ^5.0.2]

**Comment by RaiJanvi** [comment]
Try adding internet permission in Android Manifest file.
<uses-permission android:name="android.permission.INTERNET"/>

**Comment by Taym95** [comment]
> Try adding internet permission in Android Manifest file. `<uses-permission android:name="android.permission.INTERNET"/>`

Yes, you can add  `<uses-permission android:name="android.permission.INTERNET"/>`, note: this is needed only in debug!

**Comment by thusithadeepal9356** [comment]
@Taym95  @RaiJanvi  It is already added to android manifest
